cypress/integration/features/changeStatus.spec.js

```diff
@@ -0,0 +1,49 @@
+/// <reference types="cypress" />
+context('status page', () => {
+  beforeEach(() => {
+    cy.injectAxe();
```

Comment:
That's my first time seeing this! Had no idea such tools exist. Though, quick glance at tool's documentation shows that `cy.injectAxe();` has to be called after `cy.visit('url')`:

![image](https://user-images.githubusercontent.com/43747286/126506537-bc2bfd72-83a9-4cfa-a9cf-1d88152d1870.png)

I could be wrong though.

Reply:
Yup! You're right! We also need to call a command that will actually check for the accessibility violations! Thanks for flagging this!